I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 67, in script call
    $ renpy.call(target_label)
  File "game/labels/location/mountain.rpy", line 52, in script
    $ game.setVIPChanges()
  File "game/labels/location/mountain.rpy", line 52, in <module>
    $ game.setVIPChanges()
AttributeError: 'StoreModule' object has no attribute 'PP'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 67, in script call
    $ renpy.call(target_label)
  File "game/labels/location/mountain.rpy", line 52, in script
    $ game.setVIPChanges()
  File "/home/g2/Games/DefilerWings/DefilerWings-1.4.1-all-noteat2/renpy/ast.py", line 805, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/home/g2/Games/DefilerWings/DefilerWings-1.4.1-all-noteat2/renpy/python.py", line 1460, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/labels/location/mountain.rpy", line 52, in <module>
    $ game.setVIPChanges()
  File "pythoncode/game.py", line 185, in setVIPChanges
  File "pythoncode/game.py", line 31, in __init__
AttributeError: 'StoreModule' object has no attribute 'PP'

Linux-5.15.0-71-generic-x86_64-with-debian-bookworm-sid
Ren'Py 6.99.7.858
Defiler Wings 1.4.1.fdsc
